include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -41,11 +41,21 @@
 #endif
 
 #define ROM_ADDR (CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET)
+#ifdef CONFIG_TI_CCFG_PRESENT
+  #define ROM_ADDR (CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET)
+  #define ROM_SIZE (CONFIG_FLASH_SIZE*1K - CONFIG_FLASH_LOAD_OFFSET - \
+		    CONFIG_TI_CCFG_SIZE)
+
+  #define CCFG_ADDR (ROM_ADDR + ROM_SIZE)
+  #define CCFG_SIZE (CONFIG_TI_CCFG_SIZE)
```

Comment:
No, I thought initially that other TI devices shared a similar CCFG concept with different options, but I only find this for the CC13xx / CC26xx SoCs. I can remove CONFIG_TI_CCFG_SIZE and hardcode it here, if that's okay.